app/External/Stripe/SignatureValidator.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\External\Stripe;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Log;
+use Spatie\WebhookClient\SignatureValidator\SignatureValidator as SignatureValidatorBase;
+use Spatie\WebhookClient\WebhookConfig;
+
+//Set up using https://docs.stripe.com/webhooks?verify=verify-manually
+
+class SignatureValidator
+{
+    public function isValid(Request $request, WebhookConfig $config): bool
+    {
+        $signatureHeader = $request->header($config->signatureHeaderName);
+        $payload = $request->getContent();
+        $secret = $config->signingSecret;
+
+        $parsedResult = [];
+        parse_str(str_replace(',', '&', $signatureHeader), $parsedResult);
+
+        if (! array_key_exists('t', $parsedResult)) {
+            Log::info("Stripe webhook doesn't have timestamp");
```

Comment:
We probably shouldn't be logging this like this. It'll just show up in the log as that one line with no context around. Maybe this as a code comment for the developer would be better?